hieradata/class/integration/mongo.yaml

```diff
@@ -0,0 +1,7 @@
+---
+
+mongodb::s3backup::backup::s3_bucket: govuk-mongodb-backup-s3-integration
+mongodb::s3backup::backup::standalone: 'True'
+mongodb::server::s3_backups: true
+mongodb::server::replicaset_members:
+  'mongo-1':
```

Comment:
I dont think this should be here at all. It will actually break the mongo cluster.